independent-projects/qute/core/src/test/java/io/quarkus/qute/ParserTest.java

```diff
@@ -180,6 +180,18 @@ public void testCdata() {
         assertEquals("Hello world <strong>", engine.parse("Hello {name} {[<strong>]}").data("name", "world").render());
     }
 
+    @Test
+    public void testRemoveStandaloneLines() {
+        Engine engine = Engine.builder().addDefaults().removeStandaloneLines(true).build();
+        String content = "{@java.lang.String foo}\n" // -> standalone
+                + "\n"
```

Comment:
So, this appears to validate that the description:

> A *standalone line* is a line that contains only section tags (e.g. `{#each}` and `{/each}`), parameter declarations (e.g. `{@org.acme.Foo foo}`) and whitespace characters.

has a confusing `and`. I initially thought this applied to empty lines as well, but this test says it does not.

Perhaps the following is less confusing for people like me?

> A *standalone line* is a line that contains at least one of: section tags (e.g. `{#each}` and `{/each}`) or parameter declarations (e.g. `{@org.acme.Foo foo}`), ignoring any whitespace characters on the same line.

It's not perfect yet, though so if you find better, I'll take it :)

Reply:
Yeah, as I read it it's confusing... let's fix it!

Reply:
Is it better now?